legacy/src/scss/lists/_patterns_labelled-list.scss

```diff
@@ -1,6 +1,15 @@
+@import "~vanilla-framework/scss/base";
+
 @mixin maas-labelled-lists {
+  //XXX can be removed / replaced with whatever mixin will contain the placeholders %vf-list, %vf-list-item used below, once  this is fixed: https://github.com/canonical-web-and-design/vanilla-framework/issues/2808
```

Comment:
I don't think we need to make any changes to any legacy files if we're keeping vanilla at 2.0.1, right?

Reply:
It won't compile without this. (Attaching a screenshot below)